adapter/src/hermes/adapter/stdio/stdio.cc

```diff
@@ -631,9 +631,9 @@ int HERMES_DECL(fgetc)(FILE *stream) {
     auto existing = mdm->Find(stream);
     if (existing.second) {
       LOG(INFO) << "Intercept fgetc." << std::endl;
-      unsigned char value;
-      auto ret_size = read_internal(existing, &value, sizeof(value), stream);
-      if (ret_size == 1) {
+      int value;
```

Comment:
I think `fgetc`, `getc`, and `_IO_getc` were correct before in that they are only supposed to read one byte. It is only `getw` that reads `sizeof(int)` bytes.

Reply:
Reverted the changes for fgetc, getc and _IO_getc